src/System.CommandLine/ParseResult.cs

```diff
@@ -129,6 +129,14 @@ symbol switch
         public T? GetValue<T>(Option<T> option)
             => RootCommandResult.GetValue(option);
 
+        /// <summary>
+        /// Gets the parsed or default value for the specified symbol name, in the context of parsed command.
```

Comment:
this is important: the implementation provides only values for the parsed command, not for the entire symbol tree.